rest-api-spec/src/main/resources/rest-api-spec/test/mtermvectors/11_basic_with_types.yml

```diff
@@ -0,0 +1,85 @@
+setup:
+  - do:
+      indices.create:
+          index: testidx
+          body:
+            mappings:
+              testtype:
+                properties:
+                  text:
+                    type :         "text"
+                    term_vector :  "with_positions_offsets"
+  - do:
+      index:
+        index: testidx
+        type:  testtype
+        id:    testing_document
+        body:  {"text" : "The quick brown fox is brown."}
+
+  - do:
+      indices.refresh: {}
+
+---
+"Basic tests for multi termvector get":
+
+  - do:
+      mtermvectors:
+        "term_statistics" : true
+        "body" :
+          "docs":
+            -
+                "_index" : "testidx"
+                "_type" : "testtype"
+                "_id" : "testing_document"
+
+  - match: {docs.0.term_vectors.text.terms.brown.term_freq: 2}
+  - match: {docs.0.term_vectors.text.terms.brown.ttf: 2}
+
+  - do:
+        mtermvectors:
+          "term_statistics" : true
+          "body" :
+            "docs":
+              -
+                  "_index" : "testidx"
+                  "_type" : "testtype"
+                  "_id" : "testing_document"
+
+  - match: {docs.0.term_vectors.text.terms.brown.term_freq: 2}
+  - match: {docs.0.term_vectors.text.terms.brown.ttf: 2}
+
+  - do:
+        mtermvectors:
+          "term_statistics" : true
+          "index" : "testidx"
+          "body" :
+            "docs":
+              -
+                  "_type" : "testtype"
+                  "_id" : "testing_document"
+
+  - match: {docs.0.term_vectors.text.terms.brown.term_freq: 2}
+  - match: {docs.0.term_vectors.text.terms.brown.ttf: 2}
+
+  - do:
+        mtermvectors:
+          "term_statistics" : true
+          "index" : "testidx"
+          "type" : "testtype"
+          "body" :
+            "docs":
+              -
+                  "_id" : "testing_document"
+
+  - match: {docs.0.term_vectors.text.terms.brown.term_freq: 2}
+  - match: {docs.0.term_vectors.text.terms.brown.ttf: 2}
+
+  - do:
+        mtermvectors:
+          "term_statistics" : true
+          "index" : "testidx"
+          "type" : "testtype"
+          "ids" : ["testing_document"]
+
+  - match: {docs.0.term_vectors.text.terms.brown.term_freq: 2}
+  - match: {docs.0.term_vectors.text.terms.brown.ttf: 2}
```

Comment:
What is the expected behaviour when docs are stored with a custom type but queried via a typeless api?
Not sure there's a test for that here?

Reply:
👍 the tests in that PR should cover this case.